gradle/libs.versions.toml

```
@@ -10,7 +10,7 @@ lifecycleRuntimeKtx = "2.8.5"
 activityCompose = "1.9.2"
 composeBom = "2024.09.01"
 reactiveStreams = "1.0.4"
-vertexAI = "16.0.0-beta04"
+vertexAI = "16.0.0-beta05"
```

Reverting this version bump due to https://github.com/firebase/firebase-android-sdk/issues/6271

```suggestion
vertexAI = "16.0.0-beta04"
```